gemfiles/rails4.2.gemfile

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+#
+# Copyright (C) 2023  Sutou Kouhei <kou@clear-code.com>
```

Comment:
Could you use your information for what you created? :-)

Reply:
Done!